policies/experimental-features.md

```diff
@@ -0,0 +1,46 @@
+Experimental Feature Policy
+===========================
+
+A feature in OpenSSL may be designated as an experimental feature.
+
+No stability or compatibility guarantees are offered for experimental features.
+An experimental feature, or any API, protocol or file format which is part of an
+experimental feature, may be changed arbitrarily and in compatibility-breaking
+ways, or removed entirely at any time, including between releases.
+
+An experimental feature is not enabled by default and must be explicitly enabled
+at build time in order to be used. The enable flag to enable an experimental
+feature must have the form `enable-unstable-XXX`. The corresponding `ifdef`
+shall have the form `OPENSSL_NO_UNSTABLE_XXX`.
```

Comment:
I would not mix "unstable" with "experimental", they have a somewhat different meaning. Maybe I'm inspired by Debian, which does have an experimental dist that's very distinct from the unstable dist. I that sense, I would call the master branch "unstable".

So I'd rather see this stick with "experimental", including when naming config options and macros.

Reply:
I'd stick to experimental;